recipes/pyosf/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pyosf" %}
+{% set version = "1.0.4" %}
+{% set sha256 = "2c719f5c473784d27b5ba7aeb5f96dd37c377b1c0ccb1b8333f334fd2ce59d3c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-{{sha256}}.zip
+  url: https://github.com/psychopy/pyosf/archive/{{version}}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - requests
+    - pytest-runner
```

Comment:
Yeah, `pytest-runner` is often in `setup_requires` if it is used. So this makes sense to have it in `build`.

Though seeing `requests` in `build` is weird. I see that was added recently. Is there a reason for having it in `build`/`setup_requires`?

xref: https://github.com/psychopy/pyosf/pull/4